src/game/g_game.pas

```diff
@@ -5925,6 +5926,24 @@ procedure GameCVars(P: SSArray);
       end
     end;
 
+    'r_common_team_color': begin
+      if Length(P) > 1 then
+      begin
+        if (gGameSettings.GameMode = GM_TDM) or (gGameSettings.GameMode = GM_CTF) then
```

Comment:
This effectively duplicates the condition in `TPlayer.UpdateTeamColors()`. So just inline that method here.